File: MATEMÁTICA/2489/main.rb
# frozen_string_literal:true

def h(a, d, r)
  x = (90 - r) * Math::PI / 180
  a - Math.tan(x) * d
end

while (line = gets)
  a, d, r = line.split.map(&:to_f)
  puts format('%.4f', h(a, d, r))
end

File: MATEMÁTICA/2489/main.dart
import 'dart:io';
import 'dart:math';

double h(double a, double d, double r) {
  double x = (90 - r) * pi / 180;
  return a - tan(x) * d;
}

void main() {
  String? line;
  while ((line = stdin.readLineSync()) != null) {
    var [a, d, r] = line!.split(' ').map(double.parse).toList();
    print(h(a, d, r).toStringAsFixed(4));
  }
}
